lib/src/mapper/mapper.dart

```diff
@@ -18,7 +18,9 @@ class ConferenceMapper {
     var status =
         ConferenceStatus.decode(map["status"] as String? ?? "DEFAULT") ??
             ConferenceStatus.defaultStatus;
-    return Conference(alias, id, isNew, participants, status);
+    var spatialAudioStyle = SpatialAudioStyle.decode(map["spatialAudioStyle"] as String? ?? "INDIVIDUAL");
```

Comment:
Why do you need to provide a default value (ie. `?? "INDIVIDUAL"`) if the `spatialAudioStyle` is optional in the `Conference` class?

Reply:
I read in the document that this option is selected by default. But you’re right, if it’s optional and not selected then it should be nil instead of "INDIVIDUAL". I will change it.